service-registry/src/main/java/io/servicecomb/serviceregistry/RegistryUtils.java

```diff
@@ -66,7 +66,7 @@ public static void run() {
     serviceRegistry.run();
   }
 
-  public static void destory() {
```

Comment:
Need to add deprecated annotation and call the new fixed method to avoid the API broken issue.